system/modules/core/elements/ContentGallery.php

```diff
@@ -284,12 +286,15 @@ protected function compile()
 			// Do not index or cache the page if the page number is outside the range
 			if ($page < 1 || $page > max(ceil($total/$this->perPage), 1))
 			{
+				/** @var \PageModel $objPage */
```

Comment:
This global statement is actually unnecessary because `$objPage is already defined in the first line of `compile()` method.